app/models/broker_stock.rb

```diff
@@ -1,6 +1,6 @@
 class BrokerStock < ApplicationRecord
   belongs_to :broker
-  validates :name, presence: true
-  validates :quantity, presence: true
-  validates :price, presence: true
+  #   validates :companyname, presence: true
+  #   validates :quantity, presence: true
+  #   validates :price, presence: true
```

Comment:
Remove unnecessary comments

Reply:
Ma'am can we leave this comment here? I just commented it just to make the current feature I'm working on to work. We're gonna need this eventually so, I just left it there.